Cargo.toml

```diff
@@ -18,8 +18,9 @@ anyhow = "1.0.38"
 atty = "0.2"
 brotli = { version = "3.3.0", default-features = false, features = ["std"] }
 chardetng = "0.1.15"
+chrono = { version = "0.4", optional = true }
```

Comment:
What about using [`time`](https://crates.io/crates/time) crate instead of this if possible?